index.src.html

```diff
@@ -3474,6 +3552,24 @@
         {{VideoFrame/[[format]]}}, and |optLayout|.
     9. Return |combinedLayout|.
 
+: <dfn for=VideoFrame>Verify Rect Sample Alignment</dfn> (with |format| and |rect|)
+:: 1. If |format| is `null`, return `true`.
```

Comment:
@sandersdan, seem right?

Reply:
Yeah, this seemed like the lesser evil vs checking for `null` prior to the call (multiple callers). You lost me a bit on error messages; we aren't reporting an error here.